cpp/test/Ice/objects/AllTests.cpp

```diff
@@ -253,27 +253,27 @@ allTests(Test::TestHelper* helper)
     {
         LPtr v1 = std::make_shared<L>("l");
         Ice::ValuePtr v2;
-        Ice::ValuePtr v3 = initial->opValue(v1, v2);
+        Ice::ValuePtr v3 = initial->opClass(v1, v2);
         test(ICE_DYNAMIC_CAST(L, v2)->data == "l");
         test(ICE_DYNAMIC_CAST(L, v3)->data == "l");
     }
 
     {
         LPtr l = std::make_shared<L>("l");
-        Test::ValueSeq v1;
+        Test::classSeq v1;
```

Comment:
Something is wrong with this type name.

Reply:
You mean with the capitalization, or something else?

Reply:
The capitalization, see below.